packages/appstore/src/components/after-signup-flow/after-signup-flow.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from 'react';
+import { useContentFlag, useGrowthbookGetFeatureValue } from '@deriv/hooks';
+import { observer, useStore } from '@deriv/stores';
+import { Jurisdiction } from '@deriv/shared';
+
+const AfterSignupFlow = observer(() => {
+    const { client, ui, traders_hub } = useStore();
+    const { openRealAccountSignup, is_from_signup_account, setIsFromSignupAccount } = ui;
+    const { is_logged_in, has_active_real_account } = client;
+    const { setIsSetupRealAccountOrGoToDemoModalVisible } = traders_hub;
+
+    const { is_cr_demo, is_eu_demo } = useContentFlag();
+
+    const [direct_to_real_account_creation] = useGrowthbookGetFeatureValue({
+        featureFlag: 'direct-real-account-creation-flow',
+        defaultValue: false,
+    });
+
+    const [show_setup_real_or_go_demo] = useGrowthbookGetFeatureValue({
+        featureFlag: 'show_setup_real_or_go_demo',
+        defaultValue: false,
+    });
+
+    useEffect(() => {
+        if (!has_active_real_account && is_from_signup_account && is_logged_in) {
+            // move the user directly to setup real account
+            if (direct_to_real_account_creation && !show_setup_real_or_go_demo) {
```

Comment:
Why do we have this  `direct_to_real_account_creation && !show_setup_real_or_go_demo`  condition @sergei-deriv ,  In growthbook both can be true actually, and in that case both of these conditions will never meet. 🤔 
 
There is no need for &&, just the feature flag condition for each specific one is enough please correct me if I'm missing anything 🙇‍♀️ 
 
```suggestion
            if (direct_to_real_account_creation) {
```

Reply:
@mitra-deriv I did it because in this case something wrong with GB settings, because if `direct_to_real_account_creation === true` then we need to show real account creation modal, if `show_setup_real_or_go_demo === true` then we need to show modal when the user can choose stay on demo or create real account. But what we have to do if both flags are true? In my opinion, these flags are opposite.

Reply:
@sergei-deriv I get your concern but on growth for example they handle a case in which both flags are true they send 50% of the client to that screen and the other 50% to the other screen, which means both can be true but with different traffic of clients which is customized in GB itself. In that case if both true nothing get triggers , I hope I explained this in a good way

Reply:
@mitra-deriv Yeah, you are right. But what flow I have to choose, show choose modal or show real account creation modal? In my PR the user just stay on TH :)

Reply:
sorry I didn't get what you mean, in what case what you have to choose ? I mean based on the feature flag we show respective flow?

Reply:
And the logic who see what should be by Growthbook?

Reply:
@mitra-deriv if these 2 flags are true (direct_to_real_account_creation and show_setup_real_or_go_demo) what we have to do, show real account creation modal or show the modal from this PR? I made decision to not show anything, because it's opposite logic.

Reply:
we shouldn't have && it should be something like : 

  if (direct_to_real_account_creation ) > real account creation modal
  if ( show_setup_real_or_go_demo) >  show setup real or go demo modal 
  
  what to show to whom will be handled by growthbook I believe

Reply:
let's have a call I think we need to check this

Reply:
Okay per the discussion this is something CRO team will handle ( and they keep one true one false from their side on each experiment ) we keep the same condition as they want. Thank you @sergei-deriv for the time and explanation 🙇‍♀️